lms/djangoapps/discussion_api/serializers.py

```diff
@@ -191,12 +191,12 @@ class ThreadSerializer(_ContentSerializer):
     closed = serializers.BooleanField(read_only=True)
     following = serializers.SerializerMethodField()
```

Comment:
@robrap Please see reasoning comments for [pinned](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/discussion_api/serializers.py#L211) and [following](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/discussion_api/serializers.py#L222)